alpine/Dockerfile

```diff
@@ -12,7 +12,8 @@ ENV \
     LANG="C.UTF-8" \
     S6_BEHAVIOUR_IF_STAGE2_FAILS=2 \
     S6_CMD_WAIT_FOR_SERVICES_MAXTIME=0 \
-    S6_CMD_WAIT_FOR_SERVICES=1
+    S6_CMD_WAIT_FOR_SERVICES=1 \
+    S6_SERVICES_READYTIME=50
```

Comment:
IMHO this is odd to have in our images. This is an unhandled raise condition upstream.

Reply:
Discussion continued on https://github.com/home-assistant/core/pull/82527#issuecomment-1326414334.